tests/test_starfileio.py

```diff
@@ -0,0 +1,62 @@
+from unittest import TestCase
+import importlib_resources
+from pandas import DataFrame
+
+import aspire.data
+from aspire.io.starfile import Starfile, StarfileBlock, StarfileLoop
+
+
+import os.path
+DATA_DIR = os.path.join(os.path.dirname(__file__), 'saved_test_data')
+
+
+class StarfileTestCase(TestCase):
+    def setUp(self):
+        with importlib_resources.path(aspire.data, 'sample.star') as path:
+            self.starfile= Starfile(path)
```

Comment:
Space before `=`.

Reply:
Fixed in c7487a965a7c89b178f68cc1f84210f8da74b964